src/course-unit/sidebar/components/ReleaseInfoComponent.jsx

```diff
@@ -0,0 +1,29 @@
+import { useSelector } from 'react-redux';
+import { useIntl } from '@edx/frontend-platform/i18n';
+
+import { getCourseUnitData } from '../../data/selectors';
+import { getReleaseInfo } from '../utils';
+
+const ReleaseInfoComponent = () => {
+  const intl = useIntl();
+  const {
+    releaseDate,
+    releaseDateFrom,
+  } = useSelector(getCourseUnitData);
+  const releaseInfo = getReleaseInfo(intl, releaseDate, releaseDateFrom);
+
+  if (releaseInfo.isScheduled) {
+    return (
+      <span className="course-unit-sidebar-date-and-with">
+        <h6 className="course-unit-sidebar-date-timestamp m-0 d-inline">
+          {releaseInfo.releaseDate}&nbsp;
```

Comment:
Do we know if the dates get localized properly in the backend?

Reply:
The date in the backend is always in UTC. If we want it to be in the users time zone we can use `<FormattedDate />` and `<FormattedTime />` from `i18n`

Reply:
Yes, you are right 💯 
I had internal communication on this issue, now we have repeated legacy behavior. Let's take these improvements related to internationalization out of scope.